lib/pages/chat/seen_by_row.dart

```diff
@@ -27,84 +26,48 @@ class SeenByRow extends StatelessWidget {
-              timelineOverlayMessage
-                  ? Colors.white
-                  : Theme.of(context).colorScheme.primary,
-              BlendMode.srcIn,
-            ),
-            width: MessageTimeStyle.seenByRowIconSize,
-            height: MessageTimeStyle.seenByRowIconSize,
-          )
-        : seenByUsers.isEmpty
-            ? Icon(
-                Icons.done,
-                color: timelineOverlayMessage
-                    ? Colors.white
-                    : Theme.of(context).colorScheme.primary,
-                size: MessageTimeStyle.seenByRowIconSize,
-              )
-            : Icon(
-                Icons.done_all,
-                color: timelineOverlayMessage
-                    ? Colors.white
-                    : Theme.of(context).colorScheme.primary,
-                size: MessageTimeStyle.seenByRowIconSize,
-              );
-    // return Container(
-    //   width: double.infinity,
-    //   alignment: Alignment.center,
-    //   child: AnimatedContainer(
-    //     constraints:
-    //         const BoxConstraints(maxWidth: FluffyThemes.columnWidth * 2.5),
-    //     height: seenByUsers.isEmpty ? 0 : 24,
-    //     duration: seenByUsers.isEmpty
-    //         ? Duration.zero
-    //         : FluffyThemes.animationDuration,
-    //     curve: FluffyThemes.animationCurve,
-    //     alignment: controller.timeline!.events.isNotEmpty &&
-    //             controller.timeline!.events.first.senderId ==
-    //                 Matrix.of(context).client.userID
-    //         ? Alignment.topRight
-    //         : Alignment.topLeft,
-    //     padding: const EdgeInsets.only(left: 8, right: 8, bottom: 4),
-    //     child: Wrap(
-    //       spacing: 4,
-    //       children: [
-    //         ...(seenByUsers.length > maxAvatars
-    //                 ? seenByUsers.sublist(0, maxAvatars)
-    //                 : seenByUsers)
-    //             .map(
-    //               (user) => Avatar(
-    //                 mxContent: user.avatarUrl,
-    //                 name: user.calcDisplayname(),
-    //                 size: 16,
-    //                 fontSize: 9,
-    //               ),
-    //             )
-    //             .toList(),
-    //         if (seenByUsers.length > maxAvatars)
-    //           SizedBox(
-    //             width: 16,
-    //             height: 16,
-    //             child: Material(
-    //               color: Theme.of(context).colorScheme.background,
-    //               borderRadius: BorderRadius.circular(32),
-    //               child: Center(
-    //                 child: Text(
-    //                   '+${seenByUsers.length - maxAvatars}',
-    //                   style: const TextStyle(fontSize: 9),
-    //                 ),
-    //               ),
-    //             ),
-    //           ),
-    //       ],
-    //     ),
-    //   ),
-    // );
+
+    return getEventIcon(context, eventStatus, seenByUsers);
+  }
```

Comment:
What about `EventStatus.synced`

Reply:
from what I've seen `EventStatus.synced` has the same behiavor as `EventStatus.sent` it just mean that the event is synced with the server.